### ISSUE TITLE
canot edit number formatting in Word 2010 numbering configuration dialog

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4677]
**Reported by peter on 2014-12-09 07:51**
Hi devs,
It is not possible to edit number format in numbering configuration dialog. I checked older versions of nvda, it worked in 2014.2, the ms word updates in 2014.3 broke something and it does not work also in curent master branch. Steps to reproduce:
1. open word 2010
2. enable styles dialog (home>styles...
3. you are now probably in styles window. If not, then press f6 to go there
4. Press "new style" button
5. type style name, tab to "style type" and choose "list"
6. tab to format menu button, expand it and choose "numbering"
7. tab to "enter formatting for number" and check that this field is not editable. You can see there a part of document. This field contains symbolic representation of numbering format and it is possible to edit it with nvda 2014.2 and older.
regards
Peter

**Comment by Adriani90** [comment]
@peter are you still able to reproduce this issue?

**Comment by Adriani90** [comment]
Testing with NVDA 2019.1.1 and MS Word 2010, when tabbing to the edit field for formating the numbering symbols, I get following error and navigation through the field is not possible at all:

```
IO - inputCore.InputManager.executeGesture (19:59:15.089):
Input: kb(laptop):tab
ERROR - baseObject.ScriptableObject.bindGestures (19:59:15.147):
Error binding script caret_changeSelection in <NVDAObjects.Dynamic_WordDocument_WwNIAccessibleWindowNVDAObject object at 0x055F2250>
ERROR - baseObject.ScriptableObject.bindGestures (19:59:15.191):
Error binding script caret_changeSelection in <NVDAObjects.Dynamic_WordDocument_WwNIAccessibleWindowNVDAObject object at 0x055F2250>
ERROR - baseObject.ScriptableObject.bindGestures (19:59:15.233):
Error binding script caret_moveByLine in <NVDAObjects.Dynamic_WordDocument_WwNIAccessibleWindowNVDAObject object at 0x055F2250>
ERROR - baseObject.ScriptableObject.bindGestures (19:59:15.270):
Error binding script caret_moveByLine in <NVDAObjects.Dynamic_WordDocument_WwNIAccessibleWindowNVDAObject object at 0x055F2250>
ERROR - baseObject.ScriptableObject.bindGestures (19:59:15.309):
Error binding script caret_changeSelection in <NVDAObjects.Dynamic_WordDocument_WwNIAccessibleWindowNVDAObject object at 0x055F2250>
ERROR - baseObject.ScriptableObject.bindGestures (19:59:15.345):
Error binding script caret_changeSelection in <NVDAObjects.Dynamic_WordDocument_WwNIAccessibleWindowNVDAObject object at 0x055F2250>
IO - inputCore.logTimeSinceInput (19:59:15.997):
0.908 sec since input
IO - speech.speak (19:59:15.999):
Speaking [u'Modify Multilevel list  Dialogfeld  Number format\nPosition']
IO - inputCore.logTimeSinceInput (19:59:16.006):
0.916 sec since input
IO - speech.speak (19:59:16.006):
Speaking [u'Enter formatting for number:  Eingabefeld  *)']
IO - inputCore.InputManager.executeGesture (19:59:19.019):
Input: kb(laptop):leftArrow
IO - inputCore.InputManager.executeGesture (19:59:19.640):
Input: kb(laptop):rightArrow
IO - inputCore.InputManager.executeGesture (19:59:19.953):
Input: kb(laptop):downArrow
IO - inputCore.InputManager.executeGesture (19:59:20.391):
Input: kb(laptop):upArrow
```

@lukaszgo1 could the approach you used to solve the find dialog in Excel also work for this one?
cc: @leonardder I guess this is another display model implementation case. Right?

**Comment by lukaszgo1** [comment]
It isn't related to display mode at all. The error from your log looks very similar to #9250

**Comment by Adriani90** [comment]
Still reproducible in MS Office 2016 with NVDA 2024.2.
cc: @michaelDCurran

**Comment by Adriani90** [comment]
I cannot reproduce this in Office 365 with NVDA 2025.1.2 rC.
Since support for MS office 2016 and 2019 will end completely in oktober 2025, there will not be any accessibility updates to these versions. I am closing as works for me in the newer MS Office version.
If you are still having this issue in MS Office 2024 or 365, please comment and we can reopen.
